presto-main/src/main/java/com/facebook/presto/sql/planner/RowExpressionInterpreter.java

```diff
@@ -280,6 +282,14 @@ public Object visitCall(CallExpression node, Object context)
                 // do not interpret remote functions or cpp UDF on coordinator
                 return call(node.getDisplayName(), functionHandle, node.getType(), toRowExpressions(argumentValues, node.getArguments()));
             }
+            else if (implementationType.equals(REST)) {
```

Comment:
This should be reverted (we don't want to evaluate functions in the coordinator)